docs/proposals/operator-sdk-v1-migration.md

```diff
@@ -0,0 +1,285 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: crd-domain-rename
+authors:
+  - @adambkaplan
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-12-08
+last-updated: 2020-12-09
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Operator SDK v1 Migration
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions
+
+1. Do we declare the migration to SDK v1 a breaking update? Do we not support upgrades from v0.2.x
+   to v0.3.0?
+2. Should we move our base domain to `shipwright.io`?
+3. Should we use this opportunity to improve the namings of our components. Ex - rename
+   `build-operator` to `build-controller-manager`
+
+## Summary
+
+This proposal details a the requirements needed to migrate to Operator SDK v1.0 and higher.
+It also highlights other features or processes we may want to undertake in concert with this
+migration.
+
+## Motivation
+
+Operator SDK v1.0 introduced substantial changes to the way sdk-enabled repositories are
+structured. The changes are so drastic for legacy projects that the Operator SDK maintainers
+recommend a "lift and shift" approach to migration, versus an in-place update [1]. The scope
+and reach of these changes may be substantial, and it is possible that post migration we will
+not have a clean update path for existing Shipwright users.
+
+[1] https://sdk.operatorframework.io/docs/building-operators/golang/migration/#migration-steps
+
+### Goals
+
+* Identify key decisions that need to be made in the Operator SDK migration
+* Plan the implementation of the Operator SDK migration
+
+### Non-Goals
+
+* Define requirements for a migration tool if we introduce breaking API changes
+* Define a strategy for presenting install/upgrades via OperatorHub
+
+## Proposal
+
+### Project initialization
+
+Operator SDK initializes CRDs with two key flags - `--group` and `--domain`. `domain` refers to
+the DNS segment that all APIs for the operator share, and Operator SDK uses this to boostrap the
+entire project [2]. Our current domain is `build.dev`. We can keep this as is, or change it to 
+`shipwright.io` (or subdomain therein). The latter would be a breaking change, forcing current
```

Comment:
I would prefer changing it sooner than later :)

Reply:
See my comment above related to branding.